### ISSUE TITLE
Credentials via SSO token provider configuration are not supported

### ISSUE THREAD

**Issue by omus** [JuliaCloud/AWS.jl#628]
AWS replaced [AWS SSO with AWS IAM Identity Center](https://aws.amazon.com/about-aws/whats-new/2022/09/aws-iam-identity-center-apis-manage-users-groups-scale/). AWS.jl supported AWS SSO credentials via setting a profile with [`sso_start_url`](https://docs.aws.amazon.com/cli/latest/userguide/sso-configure-profile-legacy.html#sso-configure-profile-manual) but doesn't support handling [`sso-session` sections](https://docs.aws.amazon.com/cli/latest/userguide/sso-configure-profile-token.html#sso-configure-profile-token-manual).

**Comment by iamed2** [comment]
I don't believe this was related to the change to IAM Identity Center, I think it happened after that rename but separately.

The old method is still available: https://docs.aws.amazon.com/cli/latest/userguide/sso-configure-profile-legacy.html
